### DISCUSSION TITLE
Orchard setup in an AKS pod

### DISCUSSION THREAD

**Discussion by luis3199correia** [OrchardCMS/OrchardCore#9484] [General]
When I'm doing the setup in an AKS pod, I'm getting a 503 bad gateway(nginx), and I think this is a problem in my pod. So, I need to know the needed settings to my pod to not to get this error again.
AKS v1.18.14
nginx v1.19.6

**Comment by markgould** [comment]
Try setting the proxy-buffer-size to 16k in the ingress controller config map
